config.md

```diff
@@ -79,6 +79,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 * **`options`** (array of strings, OPTIONAL) Mount options of the filesystem to be used.
   * Linux: supported options are listed in the [mount(8)][mount.8] man page. Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed.
   * Solaris: corresponds to "options" of the fs resource in [zonecfg(1M)][zonecfg.1m].
+  * Windows: "ro" (readonly) is the only supported option for Windows.
```

Comment:
Without any RFC 2119 wording, this wording places no [compliance conditions][1] on either runtimes or config JSON.  I'd rather see wording like:

> * Windows: runtimes MUST support `ro`, mounting the filesystem read-only when `ro` is given.

Then:

* Windows runtimes are free to add support for additional options if they want.
* Config authors know they can rely on `ro` support in compliant Windows runtimes.

The drawback to allowing extensions is that configs that typo `rp` or some such won't always fail compliance testing.  You can cover that case with stricter-than-required validation (e.g the `--base` level in my opencontainers/image-tools#66 proposal) though, so I don't think it's worth MUSTing configs to only use `ro`.

The other platform entries for `options` aren't currently great examples of this, although I hope to revive #771 if/when opencontainers/runc#1460 lands, and that would help the Linux case.

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc5/spec.md#L40-L41